src/main/scala/com/microsoft/hyperspace/index/ExperimentalStructure.scala

```diff
@@ -0,0 +1,148 @@
+package com.microsoft.hyperspace.index
+
+import java.io.FileNotFoundException
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileStatus, Path}
+
+import com.microsoft.hyperspace.util.{JsonUtils, PathUtils}
+// scalastyle:off
+case class NewContent(root: Directory) {
```

Comment:
Why is this `NewContent` and under `ExperimentalStructure`?

Reply:
Thanks @imback82 , I have cleaned up the code. I will continue to find more places to improve, but as of now it's ready.